internal/handlers/handler_test.go

```diff
@@ -1128,6 +1128,153 @@ func TestFind(t *testing.T) {
 				`Expected a positive number in: $bitsAllClear: -1`,
 			),
 		},
+		"BitsAllSet": {
+			schemas: []string{"values"},
+			req: must.NotFail(types.NewDocument(
+				"find", "values",
+				"filter", must.NotFail(
+					types.NewDocument(
+						"name", "int32",
+						"value", must.NotFail(types.NewDocument(
+							"$bitsAllSet", int32(42))),
+					)),
```

Comment:
```suggestion
				"filter", must.NotFail(types.NewDocument( // two opening parenthesis
						"name", "int32",
						"value", must.NotFail(types.NewDocument( // two opening parenthesis
							"$bitsAllSet", int32(42),
						)), // two closing parenthesis
					)), // two closing parenthesis
```

or

```suggestion
				"filter", must.NotFail(
				    types.NewDocument(
						"name", "int32",
						"value", must.NotFail(
							types.NewDocument(
								"$bitsAllSet", int32(42),
							),
						),
					),
				),
```

not a wild mix

Reply:
Checked all tests, fixed to 2 opening/2 closing